DOL.WHD.Section14c.Test/Business/EmployerServiceTests.cs

```diff
@@ -55,5 +55,32 @@ public void ValidatesEmployer_DoesNotExistTest()
             // Assert
             Assert.IsNull(employerObj);
         }
+
+        [TestMethod()]
+        public void ValidatesEmployerTest()
+        {
+            // Arrange
+            var service = new EmployerService(_employerRepository);
+
+            // Act
+            var employerObj = service.ValidateEmployer(_employerToTest);
+
+            // Assert
+            Assert.IsTrue(employerObj);
+        }
+
+        [TestMethod()]
+        public void ValidatesEmployer_TestFail()
+        {
+            // Arrange
+            var service = new EmployerService(_employerRepository);
+            _employerToTest.PhysicalAddress.City = null;
+
+            // Act
+            var employerObj = service.ValidateEmployer(_employerToTest);
+
+            // Assert
+            Assert.IsFalse(employerObj);
+        }
```

Comment:
There also need to be validation fail tests for:
- [ ] EIN is null, empty
- [ ] LegalName is null, empty
- [ ] PhysicalAddress is null
- [ ] PhysicalAddress.StreetAddress is null, empty
- [ ] PhysicalAddress.City is null, empty
- [ ] PhysicalAddress.State is null, empty
- [ ] PhysicalAddress.ZipCode is null, empty
- [ ] PhysicalAddress.County is null, empty

Reply:
Created test for them.